frontend/app.js

```diff
@@ -210,8 +210,8 @@
                 url: "/event/:eventKey/details",
                 views: {
                     user_content: {
-                        templateUrl: "app/event/event_page.html",
-                        controller: "EventPageController as eventCtrl",
+                        templateUrl: (Utils.isMobileScreen()) ? "app/event/event_details_small_page.html" : "app/event/event_page.html",
```

Comment:
use the function that is at line 370

Reply:
@JuliePessoa in my current task I've moved that function to Utils, because I used it in other file... I don't know what is the best option for him now...

Reply:
his merge is probably happening first than yours, so it`ll be your job to change it :)